src/components/admin/InviteAdminModal/InviteAdminModal.tsx

```diff
@@ -0,0 +1,58 @@
+import { useMemo } from "react";
+import { useForm } from "react-hook-form";
+
+import { useWorldAndSpaceByParams } from "hooks/spaces/useWorldAndSpaceByParams";
+
+import { Modal } from "components/molecules/Modal";
+
+import { Button } from "../Button";
+import { Input } from "../Input";
+
+export interface InviteAdminModalProps {
+  show: boolean;
+  onHide: () => void;
+}
+
+//@debt This component is not finished yet, algoliaSearch needs to be used for search functionality.
```

Comment:
This comment might make more sense if put in another file with a constant `DISABLED_DUE_TO_` as described in https://github.com/sparkletown/sparkle/blob/redesign-2021/docs/code-style.md#maybe-could-be-useful-later-dead-code

This means there shouldn't be a comment like `// await addWorldAdmin(worldId, "userId placeholder")` as well, maybe just an empty function that has the proper parameters but has the above constant and/or comment inside.

Please check https://github.com/sparkletown/sparkle/blob/redesign-2021/src/settings/disableSettings.ts

Reply:
I understand put the component is not disabled. It just has tech debt and it needs to be finished. Adding `DISABLED_DUE_TO` would be miss-leading. I can either finish the implementation or improve the comment if you'd like.